src/Youtube2Mp3.Core/Services/IDownloadService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.IO;
+
+namespace Youtube2Mp3.Core.Services
+{
+    public interface IDownloadService
+    {
+        void DownloadMedia(Stream stream);
```

Comment:
We should probably also pass a `string path` to this method right? That way we can use it from any UI when we get going on that. I'm thinking that on the Mobile UI for instance, you have like a selection for download location or whatever, or even if you have a default download location. That can get passed to this method and it does everything for us. :)